src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1099,51 +1100,116 @@ void ApplicationManagerImpl::ReplaceHMIByMobileAppId(
 }
 
 bool ApplicationManagerImpl::StartNaviService(
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/2af4e80da9542bdd1a149ffeee0b7844a7216453.